react-app/src/components/Pagination/index.tsx

```diff
@@ -92,6 +92,7 @@ export const Pagination: FC<Props> = (props) => {
                       }
                       className="absolute w-auto h-auto opacity-0 cursor-pointer"
                       aria-labelledby="morePagesButton"
+                      data-testid="morePagesButton"
```

Comment:
I could not find a way to grab this component without `data-testid` I tried byLabelText using the aria-labelledby and getByRole with ("options") but that wasn't working for me. If you have a better way lmk ❤️

Reply:
Have you tried with just the text content?

Reply:
Yeahh, so since I am trying to grab that <select> and the only text is in the children <option> tags or in the "..." of the <div> parent it wasn't working out for me. 
I was briefly looking into trying to getting the parent and from there selecting the child, but I couldn't really find a way to do that? I could totally be missing something